client/example_object_test.go

```diff
@@ -0,0 +1,243 @@
+package client_test
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
+	"github.com/nspcc-dev/neo-go/pkg/wallet"
+	"github.com/nspcc-dev/neofs-sdk-go/client"
+	cid "github.com/nspcc-dev/neofs-sdk-go/container/id"
+	neofsecdsa "github.com/nspcc-dev/neofs-sdk-go/crypto/ecdsa"
+	"github.com/nspcc-dev/neofs-sdk-go/object"
+	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
+	"github.com/nspcc-dev/neofs-sdk-go/session"
+	"github.com/nspcc-dev/neofs-sdk-go/user"
+)
+
+// ExampleClient_ObjectGetInit demonstrates how to initialize an object retrieval
+// operation using the NeoFS SDK for Go.
+func ExampleClient_ObjectGetInit() {
+	ctx := context.Background()
+
+	// Load wallet and decrypt account
+	w, err := wallet.NewWalletFromFile("wallet.json") // path to your wallet file
```

Comment:
i'd move const wallet path/password, net/object addresses etc. to the top. And comment an invitation to set them